src/ucp/api/ucp.h

```diff
@@ -1313,7 +1313,8 @@ typedef struct ucp_worker_params {
     uint64_t                client_id;
 
     /**
-     * Align this offset of the active message data by @ref am_alignment.
+     * Align the active message data with this bytes offset.
+     * The data is aligned by @ref am_alignment.
```

Comment:
Maybe replace this sentence by an actual example:
`For example, specifying a value of 16 means offset 16 in the active message payload will be aligned to @ref am_alignment boundary. A value of 0 means the payload start address will be aligned.`

Reply:
@yosefe sorry, I went through the example and description few times and I cannot make sense out of it. I also failed to link the above to the UCT description in the ticket, so it is bit challenging to understand what problem we are trying to solve here.

Reply:
```
       This address is aligned
            │
  <-offset->│
 │          │
 ├──────────▼──────────────────────────┐
 │                                     │
 │ incoming active message payload     │
 │                                     │
 └─────────────────────────────────────┘
```

Reply:
@yosefe diagram helps. So the am_alignment alignment will start from offset X. Can you please elaborate the use case for this ?